{"errorMessage":"can't concat str to bytes","errorType":"TypeError","stackTrace":["  File \"/opt/code/localstack/localstack/services/awslambda/lambda_api.py\", line 794, in run_lambda\n    result = LAMBDA_EXECUTOR.execute(\n","  File \"/opt/code/localstack/localstack/services/awslambda/lambda_executors.py\", line 460, in execute\n    return do_execute()\n","  File \"/opt/code/localstack/localstack/services/awslambda/lambda_executors.py\", line 450, in do_execute\n    return _run(func_arn=func_arn)\n","  File \"/opt/code/localstack/localstack/utils/cloudwatch/cloudwatch_util.py\", line 170, in wrapped\n    raise e\n","  File \"/opt/code/localstack/localstack/utils/cloudwatch/cloudwatch_util.py\", line 166, in wrapped\n    result = func(*args, **kwargs)\n","  File \"/opt/code/localstack/localstack/services/awslambda/lambda_executors.py\", line 437, in _run\n    raise e\n","  File \"/opt/code/localstack/localstack/services/awslambda/lambda_executors.py\", line 433, in _run\n    result = self._execute(lambda_function, inv_context)\n","  File \"/opt/code/localstack/localstack/services/awslambda/lambda_executors.py\", line 756, in _execute\n    result = self.run_lambda_executor(lambda_function=lambda_function, inv_context=inv_context)\n","  File \"/opt/code/localstack/.venv/lib/python3.8/site-packages/localstack_ext/services/awslambda/lambda_extended.py.enc\", line 566, in run_lambda_executor\n","  File \"/opt/code/localstack/localstack/services/awslambda/lambda_executors.py\", line 662, in run_lambda_executor\n    log_output += \"\\n%s\" % additional_logs\n"]}
